README.md

```diff
@@ -1,16 +1,118 @@
 # Explore education statistics service
 [![Build Status](https://dfe-gov-uk.visualstudio.com/s101-Explore-Education-Statistics/_apis/build/status/Explore%20Education%20Statistics?branchName=master)](https://dfe-gov-uk.visualstudio.com/s101-Explore-Education-Statistics/_build/latest?definitionId=200&branchName=master)
 
+## Project structure
+
+The project is primarily composed of two areas:
+
+### Public frontend (for general public users)
+
+- **UI** 
+    - NextJS React app
+- **Content API** 
+    - Depends on the **Admin's Publisher** function to generate its cache
+    - Depends on SQLServer `content` database
+- **Data API**
+    - Requires a SQLServer database
+    - Depends on SQLServer `statistics` database
```

Comment:
Could also mention that in every other envrionment other than our local one, it's `public-statistics`. Locally Admin and Data-Api share the same database because currently without data factory we have know way of publishing data locally.